src/routes/solid-router/reference/components/hash-router.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: HashRouter
+---
+
+The `HashRouter` is a top level component that manages the routing of your application.
+It's a client side router, utilizing hash-values in the URL, allow your single page application to simulate a multi-page application.
+
+The advantage of hash-routing is that the whole application can run from a single HTML file, which can be hosted on a static file server.
+
+A disadvantage of this approach, compared to the browser-history like the [`Router`](/solid-router/reference/components/router), is that hash-routing isn't SEO friendly.
+This is because most search engines don't index the hash part of the URL, so they only see the index page of your application.
```

Comment:
```suggestion
Compared to a browser-router, such as [`Router`](/solid-router/reference/components/router), is that this approach is not SEO friendly.
Because most search engines do not index the hashed portion of a URL, they are only able to see the index page of your application when using this approach.
```